<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>1065</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trigger Wizard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Source Schema</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboSourceSchema"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target Schema</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboTargetSchema"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tables</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboTablesOpt">
        <item>
         <property name="text">
          <string>Share the same name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prefix for source tables</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suffix for source tables</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prefix for target tables</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suffix for target tables</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tableFld"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Fields</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboFieldsOpt">
        <item>
         <property name="text">
          <string>Share the same name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prefix for source fields</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suffix for source fields</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prefix for target fields</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suffix for target fields</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="attrFld"/>
      </item>
      <item>
       <widget class="QTreeView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="doSampleCheck">
        <property name="text">
         <string>Do not sample UIC</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
